test/ui/roles/edit/EditFormContainer.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import 'test/enzyme-init';
+import { mapStateToProps, mapDispatchToProps } from 'ui/roles/edit/EditFormContainer';
+import { fetchRole } from 'state/roles/actions';
+import { toggleLoading } from 'state/loading/actions';
+import { PERMISSIONS_NORMALIZED } from 'test/mocks/permissions';
+
+jest.mock('frontend-common-components', () => ({
```

Comment:
You don't need to do this, `frontend-common-components` is already globally mocked in __mocks__ directory